timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -202,6 +205,15 @@ def __init__(
         self.optimization_strategy: Optional[Literal["onnx", "openvino"]] = hyperparameters.get(
             "optimization_strategy", None
         )
+        if self.optimization_strategy is not None:
+            warnings.warn(
+                (
+                    "optimization_strategy is deprecated and will be removed in a future release. "
+                    "We recommend using Chronos-Bolt models for fast inference on the CPU."
+                ),
+                category=DeprecationWarning,
```

Comment:
Let's change this to `FutureWarning` since `DeprecationWarning` is suppressed by default